eclipse/pom.xml

```diff
@@ -33,7 +33,7 @@
 	</scm>
 
 	<properties>
-		<tycho.version>2.2.0</tycho.version>
+		<tycho.version>2.3.0</tycho.version>
 		<tycho-javadoc.version>2.2.0</tycho-javadoc.version>
```

Comment:
Not sure about this but we might want to release a new version of this plugin as well to match the tycho version again. If there are no compatibility issues, we can skip this.